Config/events.cwt

```diff
@@ -66,7 +66,7 @@ event = {
 		alias_name[trigger] = alias_match_left[trigger]
 	}
 	## cardinality = 0..1
-	fire_for_sender = no
+	fire_for_sender = bool
```

Comment:
This no/yes booleans are intetional in places that do not make sense to have it set to anything else other then the variant in the rules, the other option is default.

Reply:
I would say that the ones I changed have reasons to be set as both yes and no, depending on what the user aims to achieve.